pkg/controller/webhooks/keycloak.go

```diff
@@ -0,0 +1,78 @@
+package webhooks
```

Comment:
Maybe we could template this as well for new services? Like we do for the CRDs?

Reply:
Done